pom.xml

```diff
@@ -0,0 +1,99 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.mskcc.cmo.metadb</groupId>
+  <artifactId>master</artifactId>
+  <packaging>pom</packaging>
+  <name>MSKCC CMO MetaDB Master</name>
+  <version>0.1.0</version>
+  <description>master maven module</description>
+  <url>https://github.mskcc.org/cmo/metadb/</url>
+
+  <modules>
+    <module>persistence</module>
+    <module>service</module>
+  </modules>
+
+  <parent>
+    <groupId>org.springframework.boot</groupId>
+    <artifactId>spring-boot-starter-parent</artifactId>
+    <version>2.2.1.RELEASE</version>
+  </parent>
+
+  <repositories>
+    <repository>
+      <id>jitpack.io</id>
+      <url>https://jitpack.io</url>
+    </repository>
+  </repositories>
+
+  <properties>
+    <spring.version>5.2.6.RELEASE</spring.version>
+    <jackson.version>2.9.5</jackson.version>
+    <slf4j.version>1.7.30</slf4j.version>
+    <maven.compiler.version>1.8</maven.compiler.version>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-test</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.springframework</groupId>
+      <artifactId>spring-web</artifactId>
+      <version>${spring.version}</version>
+    </dependency>
+    <!-- java nats streaming -->
```

Comment:
Does there have to be direct dependencies on nats outside of our messaging-java dependency?

Reply:
i'll double check.. might have been left over from testing and/or the original version of the codebase I was working on